### DISCUSSION TITLE
Select to allow suggestions or tags on select mode to save as string and not array

### DISCUSSION THREAD

**Discussion by Sandilly** [avo-hq/avo#3548] [Help] [answered]
Hello, 

My schema created for a form field is a `string` but for my admin form, I would like the functionality of the tags suggestions where the user can only select one. I see that the tags field has a `mode: select`, but on save. It saves as an array. So I would have to massage it to turn it into a string on save and on show or have to change the schema to be of type array, which I am not sure if it makes much sense. 

Is there such functionality that will address this out the box, if not what is the best approach?

**Comment by Paul-Bob** [comment, marked as answer]
Hello @Sandilly.

You can achieve this behavior by using the [`update_using` field option](https://docs.avohq.io/3.0/field-options.html#parse-value-before-update) which will parse the value before saving it into the database.
